Sodium.podspec

```diff
@@ -32,5 +32,5 @@ Pod::Spec.new do |s|
 
   s.requires_arc = true
 
-  s.vendored_frameworks = "Clibsodium.xcframework"
+  s.vendored_frameworks = "Clibsodium/Clibsodium.xcframework"
```

Comment:
I'm not sure if this change to the podspec works correctly with the directory structure changes.